package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "db:start": "docker compose up -d",
     "db:stop": "docker compose down",
-    "dev": "pnpm db:start && pnpm --parallel --filter './apps/admin-dashboard' --filter './apps/api' dev",
+    "dev": "pnpm db:start && pnpm --parallel --filter './apps/admin-dashboard' --filter './apps/client-frontend' --filter './apps/api' dev",
```

Comment:
I added this so that the `dev` command from the top of the repo also runs the client frontend